project-template/values.yaml

```diff
@@ -105,7 +105,7 @@ servePublicly: true
 #  - dev
 
 # mysqlBackup:
-#   schedule: "*/1 * * * *"
+#   schedule: "30 4 * * *"
 #   # for mysql 5.7 this might be changed according to: https://github.com/worldiety/mysql-scp-backup
```

Comment:
I would make this an extra commit

Reply:
this was an unsaved changed from #34 . I just wanted to have a reasonable value in `values.yaml`